src/explore-education-statistics-frontend/.env

```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=http://localhost:5000
 FUNCTION_API_BASE_URL=http://localhost:7071
 GA_TRACKING_ID=UA-xxxxxxxxx-1
 GA_TRACKING=false
+HOTJAR_TRACKING=true
```

Comment:
Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete

Reply:
Yep merge and I'll test but yeah not going to enable for prod.